public/interviews.js

```diff
@@ -412,175 +412,273 @@ filter: drop-shadow(0px 0px 5px rgba(0, 0, 0, 0.25));"/>
+    text: `Estonian studio and session guitar virtuoso and composer [TOOMAS VANEM](https://www.facebook.com/ToomasGuitar/‚), started his music career in the 1980s.
 
 Over the years Toomas has played in multiple rock bands and projects and has remarkable career also as session musician.
 
-In 2014 Toomas Vanem wrote, recorded and produced his first solo instrumental progressive rock music album **“Toomas Vanem I”**, which features American bass legend *Stu Hamm*. The compositions on the album are harsh, but also varied and colorful. A collage of different stories / stories about the world and life written in the language of music.
+In 2014 [**Toomas Vanem**](https://www.facebook.com/ToomasGuitar/‚) wrote, recorded and produced his first solo instrumental progressive rock music album **“Toomas Vanem I”**, which features American bass legend *Stu Hamm*. 
 
-**“Toomas Vanem II”**, was released at the beginning of 2022 and it is self described as *“unique blend of styles from jazz to metal fusion music that is far beyond the limits of your imagination”*. 
-**“Toomas Vanem II”** was mixed and produced by **Toomas Vanem** and mastered by **Keijo Koppel** at **RoundSound Studios**. 
-The cover art is signed by **Merliis Järve** and the album design was realized by **Maris Savik**.
 
-In recent news, **TOOMAS VANEM** has been invited to play on festival **SÕRU JAZZ FESTIVAL**, which will be held on 18.06.2022 on the Estonian island of Hiiumaa.
+The compositions on the album are harsh, but also varied and colorful. A collage of different stories / stories about the world and life written in the language of music.
 
+**“Toomas Vanem II”**, was released at the beginning of 2022 and it is self described as *"a unique blend of styles from jazz to metal fusion music that is far beyond the limits of your imagination”*. 
 
+**“Toomas Vanem II”** was mixed and produced by [**Toomas Vanem**](https://www.facebook.com/ToomasGuitar/‚) and mastered by **Keijo Koppel** at [**RoundSound Studios**](https://de-de.facebook.com/RoundSoundStuudio). 
+The cover art is signed by [**Merliis Järve**](https://www.instagram.com/merliisart/?hl=de) and the album design was realized by [**Maris Savik**](https://www.instagram.com/preilimaris/?hl=de).
 
-*As always, a huge **Thank You** for your time, openness and availability!*
+In recent news, [**Toomas Vanem**](https://www.facebook.com/ToomasGuitar/‚) has been invited to play on festival [**SÕRU JAZZ FESTIVAL**](https://www.facebook.com/sorujazz), which will be held on 18.06.2022 on the Estonian island of Hiiumaa.
 
 
+*As always, a huge **Thank You** for your time, openness and availability!*
 
-**AAP – Hello, Toomas! Welcome on our virtual pages and thank you for taking the time to do this interview with All-Access-Pass!**
+<hr style="background-color: #dfdfdf; height: 2px;box-shadow: 0px -5px 10px rgba(0, 0, 0, 0.25); width: 100px;
```

Comment:
could you explain what hr is?

Reply:
i do not really understand how this kind of styling works, i think this is inline-styling isn't it?

Reply:
we saw you use the text:`to place your interview here with inlinestyling`.

Reply:
Thank you for the review ! Yes the data that I use is quite voluminous and the easiest way to edit my array object containing the interviews is to add the style directly to the interview texts. The <hr> for instance adds the styled divider that appears each few interviews. This all also goes hand in hand with the 'import Markdown from "markdown-to-jsx";' from the [interviewsId]. :)